.github/workflows/slack-notifier.yml

```diff
@@ -0,0 +1,31 @@
+name: IDE Experience team notifier
+run-name: Notify the IDE Experience team about relevant issues
+on:
+  issues:
+    types:
+      - labeled
+
+jobs:
+  send-slack-notification:
+    if: ${{ github.event.label.name == 'in:ide' || github.event.label.name == 'in:eclipse-plugin' || github.event.label.name == 'in:idea-plugin' || github.event.label.name == 'in:tooling-api' }}
+    runs-on: ubuntu-latest
```

Comment:
that's a docker image, right?
does it matter in anyway which image we use here?
footprint or anything?

Reply:
It's a copy-paste thing; we don't have any requirements for where to run the action.